apps/analog-alert/src/analog.c

```diff
@@ -37,9 +37,10 @@ float adc_norm;
 static struct device *gpiob;		/* GPIO device */
 static struct device *adc_dev;
 
-static void read_adc(struct knot_proxy *proxy)
+/* Write not supported */
+int write_fail(int id)
```

Comment:
What is the purpose of this function?

Reply:
That's an excellent question!

It signs that it is not possible for KNoT machine to write values to this variable.
Keeping it read only.

Reply:
Please, comment it on this function.